tests/lib/rules/jest/no-vague-titles.js

```diff
@@ -142,7 +142,53 @@ ruleTester.run('no-vague-titles', rule, {
   ],
   invalid: [
     {
-      code: "it('properly should correcly appropriate all')",
+      code:
+        "it('properly should correcly appropriate all descriptive necessary')",
```

Comment:
Why the word salad? 🤔

Reply:
Not sure what my original thinking was... probably that they all error together are listed properly in the error message. Going to leave it for now.